src/middleware/validation.middleware.js

```diff
@@ -41,6 +42,7 @@ export default () => {
             res.status(400).json(SimplifiedError);
```

Comment:
if you check this very well you will notice it has to do with the joy validation setup and if this is tampered with it will affect every one work that uses that validation. that was written by the person who configures the joi validation and not me